common/CommunityToolkit.Labs.Core.SourceGenerators/ToolkitSampleMetadataGenerator.Documentation.cs

```diff
@@ -29,6 +29,12 @@ public partial class ToolkitSampleMetadataGenerator
     private const string FrontMatterRegexSubcategoryExpression = @"^subcategory:\s*(?<subcategory>.*)$";
     private static readonly Regex FrontMatterRegexSubcategory = new Regex(FrontMatterRegexSubcategoryExpression, RegexOptions.Compiled | RegexOptions.IgnoreCase | RegexOptions.Multiline);
 
+    private const string FrontMatterRegexDiscussionIdExpression = @"^labs-discussion:\s*(?<discussionId>.*)$";
```

Comment:
Let's remove "labs" from these two new fields. This source generator will serve both Labs and the main Toolkit.